src/saveload/saveload.h

```diff
@@ -299,6 +299,7 @@ enum SaveLoadVersion : uint16 {
 	SLV_REMOVE_OPF,                         ///< 212  PR#7245 Remove OPF.
 	SLV_TREES_WATER_CLASS,                  ///< 213  PR#7405 WaterClass update for tree tiles.
 	SLV_ROAD_TYPES,                         ///< 214  PR#6811 NewGRF road types.
+	SLV_PRODUCTION_HISTORY,                 ///< 215  PR#7575 Industry production histories.
```

Comment:
Sorry I forgot this last time, there is a convention in this list to have a blank line every 5 updates. Could you add that above your entry please?

Reply:
Oh, I didn't know that. Now fixed